SIRO_README.md

```diff
@@ -9,9 +9,11 @@ Project-specific README for SIRo.
 1. Install Habitat-sim `main` branch.
     * [Build from source](https://github.com/facebookresearch/habitat-sim/blob/main/BUILD_FROM_SOURCE.md), or install the [conda nightly build](https://github.com/facebookresearch/habitat-sim#recommended-conda-packages).
         * Be sure to include Bullet physics, e.g. `python setup.py install --bullet`.
+        * Be sure not install `stable` branch of Habitat-sim since some functionality is not supported
```

Comment:
@jimmytyyang note that 3-rd step in the installation instructions is "Install Habitat-sim main branch." So this line is duplicate.

Reply:
Done, I removed this